UPGRADE.rst

```diff
@@ -114,7 +226,8 @@ v1.3.1, subject to the following:
   The room statistics are essentially unused in v1.3.1 (in future versions of
   Synapse, they will be used to populate the room directory), so there should
   be no loss of functionality. However, the statistics engine will write errors
-  to the logs, which can be avoided by setting the following in `homeserver.yaml`:
+  to the logs, which can be avoided by setting the following in
+  `homeserver.yaml`:
```

Comment:
:-1:

Reply:
I don't see the problem, am wrapping to 80 chars.